x-pack/legacy/plugins/integrations_manager/public/screens/detail/screenshots.tsx

```diff
@@ -19,15 +19,19 @@ export function Screenshots(props: ScreenshotProps) {
   const { images } = props;
 
   // for now, just get first image
-  const src = toImage(images[0].src);
+  const image = images[0];
+  const hasCaption: boolean = images[0].title ? true : false;
 
   const horizontalPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 2;
-  const bottomPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 1.75;
+  const fullVerticalPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 1.75;
```

Comment:
I don't know what `fullVerticalPadding` is vs, e.g., `verticalPadding` which is pairs with `horizontalPadding` above.

NBD. Just commenting that it's not clear to me.